plugins/outputs/influxdb/README.md

```diff
@@ -16,6 +16,7 @@ The InfluxDB output plugin writes metrics to the [InfluxDB v1.x] HTTP or UDP ser
   # urls = ["http://127.0.0.1:8086"]
 
   ## The target database for metrics; will be created as needed.
+  ## For UDP url endpoint database needs to be configured on server side.
```

Comment:
Place this above the `urls` token. Also, it may read more clearly as:
`Server must be configured to listen on a UDP socket in order for 'UDP' to work here`

Reply:
@glinton this docs change is about database configuration, if you change database parameter on the client side for UDP endpoint your data will be still written to the database that is configured on the server side. For HTTP and UNIX it is possible to change database on the client side. The only place that mentions this behavior is in this ticket [#2055](https://github.com/influxdata/telegraf/issues/2055#issuecomment-261350359)

Reply:
Ok, I understand now. It was not immediately clear that is what was meant by "configured".